### PR TITLE
feat: CLI argument '--start-hidden' to put the app in background on start

### LINKED ISSUE #654: Start minimized?
How can I start the rocketchat client on Linux in minimized to tray mode?
thank you! :)

### PR DESCRIPTION
**Pull request by lutra**
@RocketChat/electron

Closes #654

This change will fix the requested issue to start Rocket.Chat client in background immediately after starting the application.

### CODE REVIEW

File: src/main/mainWindow/index.js [hunk 1]
```diff
@@ -85,6 +85,11 @@ async function createMainWindow() {
 	if (process.env.NODE_ENV === 'development') {
 		mainWindow.webContents.openDevTools();
 	}
+
+	if (app.commandLine.hasSwitch('start-hidden')) {
```
Comment (tassoevan) [inline src/main/mainWindow/index.js]: AFAIK `app.commandLine` refers to flags passed to the Chromium processes, not being the same as `process.argv`.

**Comment by thomas-mc-work** [comment]
@tassoevan Why has this been closed?

**Comment by tassoevan** [comment]
@thomas-mc-work There was no feedback from the opener after review, nor CLA signing.

**Comment by thomas-mc-work** [comment]
Maybe one of the core contributors could jump in and add these four lines themselves? That'd be awesome!

**Comment by lutra** [comment]
> @thomas-mc-work There was no feedback from the opener after review, nor CLA signing.

Sorry for the real late latency. Now I signed the CLA and appreciate if the pull request would be opened again.

**Comment by SteeveGL** [comment]
Any update?

**Comment by iamjameswalters** [comment]
> 
> 
> Any update?

Yes, I'm quite interested in seeing this quick fix make it into release!

**Comment by dlecan** [comment]
> >    Any update?

> Yes, I'm quite interested in seeing this quick fix make it into release!

Me too, but now conflicts need to be solved :-(

@lutra?

**Comment by tfilo** [comment]
Is there any progress on solving this ?

**Comment by adam-bapm** [comment]
Following along as well.  Would love to have this ability.

**Pull request merged**